### PR TITLE
Reboot SLE Micro Configuration

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/20276
#2003 

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by 0rnela — APPROVED** [review]

**Comment by 0rnela** [comment]
@keichwa -- approved. Does it need backporting to 4.3?

**Comment by keichwa** [comment]
> @keichwa -- approved. Does it need backporting to 4.3?

Welder didn't request it.  In this regard, 4.3 isn't wrong.  This PR is a clarification or enhancement. Thus I'd avoid a backport.  But I could do it if you think it's important.

File: modules/client-configuration/pages/clients-slemicro.adoc [hunk 1]
```diff
@@ -197,3 +197,37 @@ mgr-create-bootstrap-repo
...
+
+[[clients-slemicro-reboot]]
+== Reboot {micro}
+
+{micro} is a transactional system.
+Transactional updates in general support several reboot methods.
+It is recommended to use [systemitem]``systemd`` for rebooting in systems managed by {productname}.
+Using other methods can lead to undesired behavior.
+
+When bootstraping a transactional system on {productname}, [systemitem]``systemd`` will be configured as the reboot method ([literal]``REBOOT_METHOD``), if the system is in its default configuration.
```
Comment (jcayouette) [inline modules/client-configuration/pages/clients-slemicro.adoc:211]: bootstrapping

**Review by jcayouette — CHANGES_REQUESTED** [review]

**Comment by keichwa** [comment]
> I believe it is necessary to backport it to 4.3. I apologize for not making that clear.

Thanks, noted!

**Pull request merged**